back-end/api/quickstart/helper_test.py

```diff
@@ -60,4 +60,12 @@ def get_test_post_fields(i=1, author="testauthorid"):
     'pageSize': 20,
     'commentLink': 'link to comments',
     'comments': '{ "text": "nice test" }'
-  }
\ No newline at end of file
+  }
+
+def get_test_liked_fields(author_id):
+  return {
+  'context': 'testContext',
+  'summary': 'you liked these posts',
```

Comment:
Same here: 'you liked this' is probably more appropriate since it can be a post OR a comment.